rdkcentral/Dobby
# RDK-45299: Add Apparmor contol and profile name to the device specifi…

### Description
RDK-45299: Add Apparmor contol and profile name to the device specific container config

### Test Procedure
Add to /etc/dobby.json
      "apparmor": {
        "enabled": true,
        "defaultProfile": "dobby_default"
      }
Check DobbyDaemon startup logs is /etc/dobby,json was correctly parsed. 
Run any container and check if it uses dobby_default apparmor profile.

### Type of Change

### Requires Bitbake Recipe changes?
